Practice/index.html

```diff
@@ -6,63 +6,81 @@
-
-    <script
+      To get fontAwesome to work within your applications you will need to create an account such that you can generate the CDN below: <script
       src="https://kit.fontawesome.com/324b0077c5.js" // You will need to generate your OWN kit
       crossorigin="anonymous"
-    ></script>
-
-      Please goto this website, and generate your own link:
-      https://fontawesome.com/start
+    ></script> Please goto this website, and generate your own link:
+      https://fontawesome.com/start ok
 
  -->
+
+
     <!-- 
-   Link to custom stylesheet
+   Link to custom stylesheet ok
   -->
     <link rel="stylesheet" href="./assets/css/style.css" />
   </head>
 
   <body>
-    <!-- 
-      The links / CDNS to allow the usage of Bootstrap components with interactions (like modal/ navigation bar)
-     -->
-    <script
-      src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
-      integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
-      crossorigin="anonymous"
-    ></script>
+<!--header-->
+<header>
 
-    <script
-      src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js"
-      integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN"
-      crossorigin="anonymous"
-    ></script>
 
-    <script
-      src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"
-      integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV"
-      crossorigin="anonymous"
-    ></script>
+</header>
+
+<!--hero about-->
+<div id="about">
+<div class="jumbotron con py-4">
+  <h1 class=>HEY LOOK, IT'S EVERY BOOTSTRAP WEBSITE EVER</h1>
+  <p class="">Take a look around at the same fucking bootstrap page you've seen ten million times before! Featuring all the same things as those pages!</p>
+</div>
+</div>
+
+<!--
+ <div class="container mw-100" style=" background-image: url(./assets/image/header.jpg)">
+  <h1 class="h1 text-white">HEY LOOK, IT'S EVERY BOOTSTRAP WEBSITE EVER</h1>
+  <p class="text-white">Take a look around at the same fucking bootstrap page you've seen ten million times before! Featuring all the same things as those pages!</p>
+ 
+</div> -->
+
+</section>
+
+<!--service-->
+<section id="service">
+
+
+</section>
+
+<!--portfolio-->
+<section id="portfolio">
+
+
+</section>
+
+<!--contact-->
+<section id="contact">
+
+
+</section>
+
+    <!-- 
+      The links / CDNS to allow the usage of Bootstrap components with interactions (like modal/ navigation bar) ok
+     -->
+    
 
     <!-- 
        use CSS to achieve smooth scroll  OR  CDN to smooth scroll, allows you to activate smooth scroll within the application, from this github repo
       https://github.com/cferdinandi/smooth-scroll 
      -->
-    <script src="https://cdn.jsdelivr.net/gh/cferdinandi/smooth-scroll@15/dist/smooth-scroll.polyfills.min.js"></script>
+    
     <!-- 
       Script to custom JavaScript
      -->

```

Comment:
Good Job on designing the website, though the main heading text of "HEY LOOK, IT'S EVERY BOOTSTRAP WEBSITE EVER
Take a look around at the same fucking bootstrap page you've seen ten million times before! Featuring all the same things as those pages!" has to be moved to the middle. You can use 0 margin in CSS or use the Bootstrap Classes. Though animations were not used.